src/components/build/buildQuestions/questionPanelWorkArea.tsx

```diff
@@ -177,109 +177,107 @@ const QuestionPanelWorkArea: React.FC<QuestionProps> = ({
-                    {renderCommentButton()}
-                    <Grid container direction="row" alignItems="center">
-                      <Grid container justify="center" item sm={12}>
-                        <FormControl variant="outlined">
-                          <Select
-                            className="select-question-type"
-                            disabled={locked}
-                            value={type}
-                            inputProps={{
-                              name: 'age',
-                              id: 'age-native-simple',
-                            }}
-                            onChange={(e) => {
-                              props.setQuestionType(parseInt(e.target.value as string) as QuestionTypeEnum);
-                            }}
-                          >
-                            {
-                              typeArray.map((typeName, i) => {
-                                const type = QuestionTypeObj[typeName] as QuestionTypeEnum;
-                                return (
-                                  <MenuItem key={i} value={type}>
-                                    {SplitByCapitalLetters(typeName)}
-                                  </MenuItem>
-                                )
-                              })
-                            }
-                          </Select>
-                        </FormControl>
-                      </Grid>
+        <Grid container direction="row" className="build-question-column">
+          <Grid container item xs={4} sm={3} md={3} alignItems="center" className="parent-left-sidebar">
+            <div className="left-sidebar">
+              <ReactSortable
+                list={componentTypes}
+                group={{ name: "cloning-group-name", pull: "clone" }}
+                setList={setComponentType} sort={false}
+              >
+                <DragBox
+                  locked={locked}
+                  name="T"
+                  label="TEXT"
+                  className="text-box"
+                  hoverMarginTop="-0.85vw"
+                  value={QuestionComponentTypeEnum.Text}
+                />
+                <DragBox
+                  locked={locked}
+                  name="“ ”"
+                  label="QUOTE"
+                  hoverMarginTop="-1.5vw"
+                  value={QuestionComponentTypeEnum.Quote}
+                />
+                <DragBox
+                  locked={locked}
+                  name="jpg"
+                  label="IMAGE"
+                  value={QuestionComponentTypeEnum.Image}
+                />
+                <DragBox
+                  locked={locked}
+                  isImage={true} src="/images/soundicon.png"
+                  label="SOUND"
+                  value={QuestionComponentTypeEnum.Sound}
+                />
+                <DragBox
+                  locked={locked}
```

Comment:
too much changes in tsx files

Reply:
just removed 1 extra grid div from parent, it was use less